### DISCUSSION TITLE
Filename collections

### DISCUSSION THREAD

**Discussion by eddik1970** [meisnate12/Plex-Meta-Manager#649] [Q&A] [answered]
I am looking to make a Collection of first episodes of TV pilots. 

If I could create a collection where filename contains S01E01 that would be perfect? 

Is this possible?

**Comment by meisnate12** [comment, marked as answer]
Use `plex_pilots: true` with `collection_level: episode`

**Reply by eddik1970** [reply]
Did you add this yesterday? 

I found the documentation at https://github.com/meisnate12/Plex-Meta-Manager/wiki/Plex-Builders 

Thank you!
